modules/ensemble/lib/layout/list_view.dart

```diff
@@ -133,22 +124,16 @@ class ListViewController extends BoxLayoutController {
 
   bool? shrinkWrap;
 
-  bool enableSwipe = false;
-  double swipeThreshold = 0.2;
-  dynamic swipeWidget;
-  int swipeAnimationDuration = 200;
-  EnsembleAction? onSwipe;
-
   void _bind(ListViewState state) {
     widgetState = state;
   }
 }
 
 class ListViewState extends EWidgetState<ListView>
-    with TickerProviderStateMixin, TemplatedWidgetState, HasChildren<ListView> {
+    with TemplatedWidgetState, HasChildren<ListView> {
```

Comment:
Just to confirm, there is no code changes in list_view right? This is just a revert to the original version?

Reply:
Yes that is outdated, there is no changes in listView.